port/unix/omrsysinfo.c

```diff
@@ -1670,6 +1687,8 @@ omrsysinfo_get_s390_processor_feature_name(uint32_t feature)
 		return "vec_e2";
 	case OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL_ENHANCEMENT_FACILITY:
 		return "vec_pde";
+        case OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL_ENHANCEMENT_FACILITY_2:
```

Comment:
This case statement doesn't appear to be indented properly.

Reply:
Fixed in 96ffcc84dced3cdde345a701b87d88e62b072433